src/simulation/sensors/simpleVoltEstimator/simpleVoltEstimator.cpp

```diff
@@ -27,11 +27,20 @@ SimpleVoltEstimator::SimpleVoltEstimator()
 {
```

Comment:
First line of this commit is too long and the commit header runs over in the GitHub commit presentation